### PR TITLE
Add cssls description

### PR DESCRIPTION
**Pull request by nabezokodaikon**
Added supplement to cssls description

### CODE REVIEW

**Comment by mjlbach** [comment]
You should add a note on why to enable snippet support (is there any particular reason as to why this differs from what is already in the wiki?) and not include the commit to CONFIG.md

**Comment by nabezokodaikon** [comment]
Current description of cssls This is because cssls does not work with `require'lspconfig'.cssls.setup{}`.

**Comment by mjlbach** [comment]
These are the lines you should copy: https://github.com/neovim/nvim-lspconfig/blob/f9785053a4ef4aaa2d0aac958bc09a1a289d2fbf/lua/lspconfig/html.lua#L32-L34


Please add the above, squash, and remove the commited changes to CONFIG.md (that file is auto-generated, and your changes in css lua file will be enough).

File: lua/lspconfig/cssls.lua [hunk 1]
```diff
@@ -27,6 +27,19 @@ https://github.com/vscode-langservers/vscode-css-languageserver-bin
 ```sh
 npm install -g vscode-css-languageserver-bin
 ```
+
+Neovim does not currently include built-in snippets. `vscode-html-languageserver` only provides completions when snippet support is enabled.
```
Comment (mjlbach) [inline lua/lspconfig/cssls.lua:31]: This need to be changed from html to vscode-css-languageserver

File: CONFIG.md [hunk 2]
```diff
@@ -418,7 +418,7 @@ npm install -g vscode-css-languageserver-bin
 
 
 ```lua
-require'lspconfig'.cssls.setup{}
+require'lspconfig'.cssls.setup {}
```
Comment (mjlbach) [inline CONFIG.md:421]: remove this change and then squash

**Comment by nabezokodaikon** [comment]
It's complicated, so start over.

**Comment by mjlbach** [comment]
for future reference, `git rebase -i master`, change every commit but the first one from `pick` to `squash`

**Pull request closed without merging**